src/cow-react/modules/limitOrders/pure/ReceiptModal/FilledField.tsx

```diff
@@ -1,83 +1,19 @@
 // Code based on https://github.com/cowprotocol/explorer/blob/develop/src/components/orders/FilledProgress/index.tsx
 import * as styledEl from './styled'
 import { ParsedOrder } from '@cow/modules/limitOrders/containers/OrdersWidget/hooks/useLimitOrdersList'
-import { CurrencyAmount, Token } from '@uniswap/sdk-core'
-import { OrderKind } from '@cowprotocol/cow-sdk'
-import { BigNumber } from 'bignumber.js'
-import JSBI from 'jsbi'
 import { TokenAmount } from '@cow/common/pure/TokenAmount'
+import { getFilledAmounts } from '@cow/modules/limitOrders/utils/getFilledAmounts'
 
 interface Props {
   order: ParsedOrder
-  sellAmount: CurrencyAmount<Token>
-  buyAmount: CurrencyAmount<Token>
 }
 
-// TODO: using .toNumber() we potentially lose accuracy
-// TODO: must be refactored with replacing bignumber.js by @ethersproject/bignumber
-function legacyBigNumberToCurrencyAmount(currency: Token, value: BigNumber | undefined): CurrencyAmount<Token> {
```

Comment:
Moved all of this code to a separate util function `src/cow-react/modules/limitOrders/utils/getFilledAmounts.ts` because we also need the same values in execution popup, so we can reuse it.

Reply:
Great to see this code moved to an util.

Why this util is inside `limitOrders` module? isn't generic for anything that has to do with orders?